WordPress/Sniffs/Theme/NoHardCodedUrlsSniff.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * WordPress_Sniffs_Theme_NoHardCodedUrlsSniff File Doc Comment
+ *
+ * @category WordPress_Sniffs_Theme_NoHardCodedUrlsSniff
+ * @package  PHP_Codesniffer
+ * @author   khacoder
+ * @license  http://www.gnu.org/copyleft/gpl.html GNU General Public License
+ * @link     http://kevinsspace.ca
+ */
+
+/**
+ * WordPress_Sniffs_Theme_NoHardCodedUrlsSniff Class Doc Comment
+ *
+ * WARNING Verify that no hard-coded URLs are found. Manual verification necessary.
+ * An exception is made for the Author URI and (Theme) URI as set in the style.css header
+ * as well as links to wordpress.org. Links in the text of PHP/JS comments should be
+ * excluded from this check.
+ *
+ * @category WordPress_Sniffs_Theme_NoHardCodedUrlsSniff
+ * @package  PHP_Codesniffer
+ * @author   khacoder
+ * @license  http://www.gnu.org/copyleft/gpl.html GNU General Public License
+ * @link     http://kevinsspace.ca
+ */
```

Comment:
`WordPress_Sniffs_Theme_NoHardCodedUrlsSniff.` should be sufficient for the doc comment title.

@jrfnl What do you think having the `@category` as `WordPress_Theme`?

You can use your full name with an email address `Kevin Archibald <kevin@archibald.com>` if you want.

The `@link` used to link to related documentation.

As the licence of the project is MIT. The licence of the class cannot be GPL. I think the licence is not needed as we have a licence for the whole project.